package com.rakovets.course.java.core.practice.exception_handling.task_3;

import com.rakovets.course.java.core.practice.exception_handling.task_3.exception.NumberException;

public class Task {
    public static int getNumber() throws NumberException {
        return 5;
    }
}
